fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1735,7 +1735,7 @@ void checkTestConf(const char* testFile, TestConfig* testConfig) {
 			testConfig->isFirstTestInRestart = true;
 		}
 		if (attrib == "disableTss") {
-			sscanf(value.c_str(), "%d", &testConfig->disableTss);
+			testConfig->disableTss = strcmp(value.c_str(), "true") == 0;
```

Comment:
It seems we have an inconsistency with how we specify boolean options (see `configureLocked` for an example of using an integer). It doesn't have to be done here, but perhaps we could extract this into a parse boolean function that will except either 1/0 or true/false.

Reply:
+1

Reply:
I see, nice catch, 
and I found it's also an error on master where we have `configureLocked = true`
https://github.com/apple/foundationdb/blob/master/tests/fast/ConfigureLocked.toml#L2